Makefile

```
@@ -53,7 +53,7 @@ bench:
 # firstword, and word select indexes from said array.
 release/goss-%: $(GO_FILES)
 	CGO_ENABLED=0 GOOS=$(firstword $(subst -, ,$*)) GOARCH=$(word 2, $(subst -, ,$*)) go build -ldflags "-X main.version=$(TRAVIS_TAG) -s -w" -o $@ $(exe)
-	sha256sum $@ > $@.sha256
+	sha256sum $@ | sed 's/release\/goss/goss/g' > $@.sha256
```

Wonder if this would work to avoid sed
```
cd $(@D) && sha256sum $(@F)
```

---

Well, to be really honest I have no clue about your suggestion! :joy: Lacking Make knowledge here.

But I totally understand (and agree) with removing the sed operation, so I'll use your suggestion ;)

---

So, just to understand, the "solution" would be:

```suggestion
cd $(@D) && sha256sum $(@F) > $(@D).sha256sum
```
?